rsmtool/notebooks/summary/evaluation.ipynb

```diff
@@ -39,11 +39,33 @@
     "            df_eval['system score type'] = 'scale' if config.get('use_scaled_predictions') == True or config.get('scale_with') is not None else 'raw'        \n",
     "            \n",
     "            # get truncation ranges\n",
-    "            trim_min = float(config.get('trim_min'))\n",
-    "            trim_max = float(config.get('trim_max'))\n",
-    "            trim_tolerance = config.get('trim_tolerance', 0.49998)\n",
-    "            min_score = trim_min - trim_tolerance\n",
-    "            max_score = trim_max + trim_tolerance\n",
+    "            # first let's see if we have them in the config file\n",
+    "            \n",
+    "            try: \n",
+    "                trim_min = float(config.get('trim_min'))\n",
+    "                trim_max = float(config.get('trim_max'))\n",
+    "                trim_min_known = True\n",
+    "            except TypeError: \n",
+    "                # if these are Nones, get them from postprocessing params\n",
+    "                postproc_params_file = os.path.join(csvdir, '{}_postprocessing_params.{}'.format(model_id,\n",
+    "                                                                                                 file_format))\n",
+    "                if os.path.exists(postproc_params_file):\n",
+    "                    df_postproc_params = DataReader.read_from_file(postproc_params_file)\n",
+    "                    trim_min = df_postproc_params['trim_min'].values[0]\n",
+    "                    trim_max = df_postproc_params['trim_max'].values[0]\n",
+    "                    trim_min_known = True\n",
```

Comment:
... and get rid of this one.

Reply:
What about something like this instead?

```
    "            # truncation ranges\n",
    "            # get the trim tolerance, min, and max from the config\n",
    "            trim_tolerance = config.get('trim_tolerance', 0.49998)\n",
    "            trim_min = config.get('trim_min')\n",
    "            trim_max = config.get('trim_max')\n",
    "            # if the trim min and max are not None, then use them to calculate min and max scores\n",
    "            if trim_min is not None and trim_max is not None:\n",
    "                min_score = trim_min - trim_tolerance\n",
    "                max_score = trim_max + trim_tolerance\n",
    "            # otherwise, use the postprocessing parameters file, if it exists, or default min and max to '?'\n",
    "            else:\n",
    "                postproc_file = os.path.join(csvdir, '{}_postprocessing_params.{}'.format(model_id, file_format))\n",
    "                if os.path.exists(postproc_file):\n",
    "                    df_postproc = DataReader.read_from_file(postproc_file)\n",
    "                    min_score = df_postproc['trim_min'].values[0] - trim_tolerance\n",
    "                    max_score = df_postproc['trim_max'].values[0] + trim_tolerance\n",
    "                else:\n",
    "                    min_score, max_score = '?', '?'\n",
```

Reply:
Don't you need to convert `trim_min` and `trim_max` to floats first?

Reply:
I'll add that, just to be safe. (Although, I thought these would always be numeric or null by the time they were saved to the JSON.)